AI/pyneogame/gym.py

```diff
@@ -26,7 +26,7 @@
 
 class Gym:
 
-    def __init__(self, player, opponent):
+    def __init__(self, player, opponent, name="autosave_model.h5"):
```

Comment:
Change class attribute 'name' to  'filename' or similar and add checkpoints with suffix '_chkpt<number>'